R/env.R

```diff
@@ -54,13 +54,13 @@
 #' env(fn)
 #'
 #'
-#' # new_env() creates by default an environment whose parent is the
```

Comment:
I'd prefer to keep this as `new_env()`. It's not perfect because it's not S3, but I think it's better than using a different word